### DISCUSSION TITLE
Shuffle All, save track

### DISCUSSION THREAD

**Discussion by ipslfn** [gateship-one/odyssey#217] [Q&A]
Hi!
I like the app there is just one issue I have.
So, when you open the app up, the right most menu by default is for songs (all tracks). You hit a big red button and this plays the first track in alphabetical order.
I'm kinda particular in what I want, which is I just want the whole library on my device to play on shuffle.
The issues I have with Odyssey is that it does not start on shuffle when hitting play in this way and it does not save the track I was listening to should the application be closed. 
With that being said, I do really love the app, it works better than all the alternatives on f-droid but it does not have this one particular functionality that other apps do (Vinyl, Phonograph, et al.) 
I am just curious if maybe I'm missing something in settings to make it perform more in line with what I'm looking for.
Thank you and I appreciate all the hard work put in to Odyssey!
![example](https://user-images.githubusercontent.com/67012312/113527070-517b6980-958a-11eb-9b33-e3b8b4a7257a.png)

**Comment by gnome17** [comment]
Sorry for the late reply ;)

Play with shuffle is currently not implemented but you could shuffle the playlist afterwards. Also odyssey does save your current playlist state. So if this doesn't work for you there seems to be an issue because it should save the state after playback is stopped.

We think about it if we integrate shuffle into the play button functionality.

**Comment by ipslfn** [comment]
Thank you! Yes I had an older android device so I figured some of the issues I had were possibly related to that. 
Thank you for the answer and I'll give it a shot when I get a new device.
